extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/DebeziumCdcIntegrationTest.java

```diff
@@ -50,6 +51,9 @@
 
     @Test
     public void mysql() throws Exception {
+        Assume.assumeFalse("https://github.com/hazelcast/hazelcast-jet/issues/2623",
+                System.getProperty("java.version").equals("15.0.1"));
```

Comment:
I don't fully understand why this error we are addressing happens with JDK 15 (and why it doesn't with JDK 11 too), but I suspect it would happen with any sub-version of JDK 15, so maybe make this check less specific? Check only the major version?

Reply:
originally I hoped that we will fix it during Jet 4.4 but we cannot be sure. I'll change it to `System.getProperty("java.version").startsWith("15")`